cmd/ops_agent_windows/run_windows.go

```diff
@@ -61,6 +62,14 @@ func (s *service) Execute(args []string, r <-chan svc.ChangeRequest, changes cha
 		return false, 2
 	}
 	s.log.Info(EngineEventID, "generated configuration files")
+
+	if err := s.runStartupChecks(); err != nil {
+		s.log.Error(1, fmt.Sprintf("failed while running startup checks: %v", err))
+		// 2 is "file not found"
+		return false, 2
+	}
+	s.log.Info(EngineEventID, "startup checks finished")
```

Comment:
Can we have a similar message for Linux?

Reply:
We could, but not we don't have any other similar messages in the Linux version. WDYT @jinghan-ma ?

Reply:
I think even though we don't currently have similar messages in Linux, this is probably something worthwhile to log for Linux as well.

I think it makes sense to add this for both Windows and Linux